packages/camera/camera_windows/lib/camera_windows.dart

```diff
@@ -0,0 +1,424 @@
+  }
+
+  @override
+  Future<void> initializeCamera(
+    int cameraId, {
+    ImageFormatGroup imageFormatGroup = ImageFormatGroup.unknown,
+  }) async {
+    final int requestedCameraId = cameraId;
+
+    /// Creates channel for camera events.
+    _cameraChannels.putIfAbsent(requestedCameraId, () {
+      final MethodChannel channel =
+          MethodChannel('flutter.io/cameraPlugin/camera$requestedCameraId');
+      channel.setMethodCallHandler(
+        (MethodCall call) => _handleCameraMethodCall(call, requestedCameraId),
+      );
+      return channel;
+    });
+
+    final Map<String, double>? reply;
+    try {
+      reply = await _pluginChannel.invokeMapMethod<String, double>(
+        'initialize',
+        <String, dynamic>{
+          'cameraId': requestedCameraId,
+        },
+      );
+    } on PlatformException catch (e) {
+      throw CameraException(e.code, e.message);
+    }
+
+    _cameraEventStreamController.add(
+      CameraInitializedEvent(
+        requestedCameraId,
+        reply!['previewWidth']!,
+        reply['previewHeight']!,
+        ExposureMode.auto,
+        false,
+        FocusMode.auto,
+        false,
+      ),
+    );
+  }
+
+  @override
+  Future<void> dispose(int cameraId) async {
+    await _pluginChannel.invokeMethod<void>(
+      'dispose',
+      <String, dynamic>{'cameraId': cameraId},
+    );
+
+    // Destroy method channel after camera is disposed to be able to handle last messages.
+    if (_cameraChannels.containsKey(cameraId)) {
+      final MethodChannel? cameraChannel = _cameraChannels[cameraId];
+      cameraChannel?.setMethodCallHandler(null);
+      _cameraChannels.remove(cameraId);
+    }
+  }
+
+  @override
+  Stream<CameraInitializedEvent> onCameraInitialized(int cameraId) {
+    return _cameraEvents(cameraId).whereType<CameraInitializedEvent>();
+  }
+
+  @override
+  Stream<CameraResolutionChangedEvent> onCameraResolutionChanged(int cameraId) {
+    /// Windows API does not automatically change the camera's resolution
+    /// during capture so these events are never send from the platform.
+    /// Support for changing resolution should be implemented, if support for
+    /// requesting resolution change is added to camera platform interface.
+    ///
+    /// Returns empty stream.
+    return const Stream<CameraResolutionChangedEvent>.empty();
+  }
+
+  @override
+  Stream<CameraClosingEvent> onCameraClosing(int cameraId) {
+    return _cameraEvents(cameraId).whereType<CameraClosingEvent>();
+  }
+
+  @override
+  Stream<CameraErrorEvent> onCameraError(int cameraId) {
+    return _cameraEvents(cameraId).whereType<CameraErrorEvent>();
+  }
+
+  @override
+  Stream<VideoRecordedEvent> onVideoRecordedEvent(int cameraId) {
+    return _cameraEvents(cameraId).whereType<VideoRecordedEvent>();
+  }
+
+  @override
+  Stream<DeviceOrientationChangedEvent> onDeviceOrientationChanged() {
+    // TODO(jokerttu): Implement device orientation detection.
+    //
+    // Returns empty stream.
```

Comment:
If the comment is just the single line of code it's commenting reworded as English, it's generally better to just omit the comment.